### ISSUE TITLE
How to rotate the page?

### ISSUE THREAD

**Issue by MarcusVNJ** [espresso3389/pdfrx#221]
I'm trying to rotate a specific page in the document (PDF), but I haven't found a clear way to do it in the implementation or in the example shown in the project demo.

**Comment by MarcusVNJ** [comment]
Sorry, I misinterpreted this library. It's not a PDF manipulator, but a viewer.

**Comment by espresso3389** [comment]
Now, git master supports rotation of the pages and also encoding PDF file:

https://github.com/espresso3389/pdfrx/blob/master/doc/PDF-Page-Manipulation.md#page-rotation

**Comment by espresso3389** [comment]
Page rotation support is now available in pdfrx 2.2.9! 

You can rotate pages when manipulating PDF documents using the `PdfPageWithRotationExtension` methods. Here's how:

```dart
final doc = await PdfDocument.openFile('document.pdf');

// Use the rotated page in page manipulation
doc.pages = [
  doc.pages[0],                 // Page 1 with original rotation
  doc.pages[1].rotatedCW90(),   // Page 2 rotated 90° clockwise
  doc.pages[2].rotatedCCW90(),  // Page 3 rotated 90° counter-clockwise
  doc.pages[3].rotated180(),    // Page 4 rotated 180°
];

// Encode the modified PDF
final pdfBytes = await doc.encodePdf();
```

For more details and complete examples, please see the [Page Rotation documentation](https://github.com/espresso3389/pdfrx/blob/master/doc/PDF-Page-Manipulation.md#page-rotation).
